src/components/organisms/plans/plans.scss

```
@@ -0,0 +1,17 @@
+@use "../../variables" as *;
+.row {
+  gap: 15px;
+  flex-wrap: nowrap;
+  justify-content: center;
+  @media screen and (max-width: $breakpoint-md) {
+    flex-direction: column;
+  }
+}
+.plans-button {
+  display: flex;
+  justify-content: center;
+  margin-top: 40px;
+  @media screen and (max-width: $breakpoint-md) {
+    margin-top: 0;
+  }
+}

```

```suggestion
.plans {
  .row {
    gap: 15px;
    flex-wrap: nowrap;
    justify-content: center;
    @media screen and (max-width: $breakpoint-md) {
      flex-direction: column;
    }
  }
  .button {
    display: flex;
    justify-content: center;
    margin-top: 40px;
    @media screen and (max-width: $breakpoint-md) {
      margin-top: 0;
    }
  }
}
```
この書き方だと、他のコンポーネントでrowを使っていた場合、そちらにも適用されてしまいます。
予想外の副作用を防ぐため、基本的にhtmlの構造をscssにも反映させて、影響範囲を狭くすることを心がけるのがおすすめです。